app/javascript/components/dashboard/ContentExistingTable.vue

```diff
@@ -194,7 +194,7 @@ export default {
     },
 
     editPath (commitment) {
-      return `/commitments/${commitment.id}/edit`;
+      window.location.replace(`/commitments/${commitment.id}/edit`);
```

Comment:
If we're keeping this (i.e. not removing Turbolinks altogether), then maybe rename this `goToEditPath` to keep it in line with the methods in ContentExisting.vue and Hero.vue